main/main.ino

```diff
@@ -984,6 +987,10 @@ void setup_wifi() {
 #  if defined(ESP32) && defined(ZgatewayBT)
     if (failure_number_ntwk > maxConnectionRetryWifi && lowpowermode)
       lowPowerESP32();
+#  else
+    if (failure_number_ntwk > maxRetryWatchDog) {
+      watchdogReboot(2);
+    }
 #  endif
   }
```

Comment:
I think
```
    if (failure_number_ntwk > maxRetryWatchDog && !lowpowermode) {
      watchdogReboot(2);
    }
```

Should be after  the endif at line 994, so that this function is used when having an ESP32 + BT, we can just add `!lowpowermode` into it. But this will be only for comfort due to the fact that the ESP32 should not go this far if it has lowpowermode activated

Reply:
Thinking this thru

Without BT enabled

1 - If low power mode is enabled (1 or 2 ), and Wifi is down, the unit will never reboot

2 - If low power mode is not enabled (0), and Wifi is down, the unit will reboot after maxRetryWatchDog

With BT Enabled

1 - If low power mode is enabled (1 or 2 ), and Wifi is down, the unit will not reboot, but will go to sleep after maxConnectionRetryWifi

2 - If low power mode is not enabled (0), and Wifi is down, the unit will reboot after maxRetryWatchDog

With lowpowermode mode the unit will never reboot due it bad wifi, but this patch does not change the existing behaviour in this situation.  So makes sense